jina/drivers/rank.py

```diff
@@ -28,11 +29,12 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.recursion_order = 'post'
 
-    def _apply_all(self, docs: 'jina_pb2.Document', context_doc: 'jina_pb2.Document', *args, **kwargs):
+    def _apply_all(self, docs: List['jina_pb2.Document'], context_doc: 'jina_pb2.Document', travers_on: str, *args, **kwargs):
         """
 
         :param docs: the chunks of the ``context_doc``, they are at depth_level ``k``
         :param context_doc: the owner of ``docs``, it is at depth_level ``k-1``
+        :param traverse_on: "matches" or "chunks"
```

Comment:
same here, if you don't need in the function, no need to add it

Reply:
Yes, u r right. The codes need clean up.